packages/glib/build.sh

```diff
@@ -67,11 +66,62 @@ termux_step_host_build() {
 termux_step_pre_configure() {
 	# glib checks for __BIONIC__ instead of __ANDROID__:
 	CFLAGS+=" -D__BIONIC__=1"
+	_PREFIX=$TERMUX_PKG_TMPDIR/prefix
+	local _WRAPPER_BIN="${TERMUX_PKG_BUILDDIR}/_wrapper/bin"
+	rm -rf $_PREFIX $_WRAPPER_BIN
+	mkdir -p $_PREFIX $_WRAPPER_BIN
```

Comment:
Quotes here please on every variable usage/assignment.

Reply:
> as this approach would conflict with the one contained as part of https://github.com/termux/termux-packages/pull/20513.

No conflict here.
This PR removes `glib` dependency on `gobject-introspection`, but keeps `gobject-introspection` dependency on `glib`. So no more cyclic dependency here.

Reply:
It happens to be a cyclic conflict, fittingly enough.
As #20513 in its current iteration solves the conflict by
building a "virtual" `gobject-introspection` to use for compiling `glib`.

The main benefit with your approach in that we don't need to invent a whole new category of "virtual" packages and muck around with the build order system to solve the issue.